src/pygambit/game.pxi

```diff
@@ -336,6 +336,39 @@ class Game:
         g.title = title
         return g
 
+    def to_arrays(self) -> typing.List[np.array]:
```

Comment:
Is it worth allowing a `dtype` argument, so the output can be in the user's desired datatype directly?  The current implementation always produces `pygambit.Rational`.  Although we're always going to be quite strict about our representation of numbers, a common use-case for converting to and from `numpy` is to use floating-point numbers.  We could take the view that once someone exports data from `pygambit`, it's on them if they encounter problems with rounding errors!

One possibility would be to support `dtype` as an argument with `float` or `fractions.Fraction` as valid types (noting that `pygambit.Rational` is actually derived from `fractions.Fraction`).  We could also support directly the `numpy` floating-point types.

Reply:
I agree, enriching the functionality is user-friendly.

If allowing a dtype argument, how do you see it working?
If dtype=float, we could convert every payoff to float and round (e.g., 2 or 3 decimals).
If dtype=fractions.Fraction, then we could convert every payoff to fraction as well.

Is dtype meant to be a simple string or something more complex?

Reply:
`fractions.Fraction` is the default (or should be), because that's the format we use internally for game data.  If floats were requested, we'd use `float(payoff)`.

I would suggest that for dtype we pass the type itself, that is, `dtype=float` or `dtype=fractions.Fraction`.  Then we could write e.g. `dtype(payoff)` - so basically if `dtype` is a type which can convert a `Fraction` to itself, then it will work, otherwise it would generate an error.

Reply:
Adjusted the code accordingly and added two tests (for float and fraction). Please let me know what you think